src/components/Header.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { View, StyleSheet, Image } from 'react-native';
+import { spacing } from '@/theme';
+
+export const Header = () => {
+  return (
+    <View style={styles.header}>
+      <Image
+        source={require('@/assets/logo/logo.png')}
```

Comment:
Change requires with imports (at the top of the file) this may improve performance.
Do this for all image require on the PR